src/app/base/components/NodeActionMenuGroup/index.scss

```diff
@@ -11,11 +11,15 @@
     }
 
     .p-button, .p-switch {
-      margin: $spv--x-small 0;
-      padding: $spv--x-small $spv--large;
+      margin: calc(0.4rem - 1px) 0;
```

Comment:
Do you remember why you chose this specific value? I wonder if we could use an existing variable here instead.

This would help stay in sync with Vanilla when there are changes to spacing and/or other styles in the future.

Reply:
my reaction to that
![image](https://user-images.githubusercontent.com/35104482/228523640-08c35476-306b-4fa2-aed8-6828e2e2bb83.png)

I'll get that committed, cheers!

Reply:
Actually - I've just committed that, and the action button group looks too thin now:
![image](https://user-images.githubusercontent.com/35104482/228524149-7a76b92f-72aa-4568-8b37-10d155b8ac1d.png)

Will investigate

Reply:
![image](https://user-images.githubusercontent.com/35104482/228524952-07d5433a-a50f-4eee-bd79-a952fd33afc5.png)

Looks like the padding comes out as `0` for some reason...